modules/core/src/smithy4s/internals/DocumentDecoderSchemaVisitor.scala

```diff
@@ -315,7 +312,7 @@ class DocumentDecoderSchemaVisitor(
     ) => Unit = {
       val jLabel = jsonLabel(field)
 
-      val maybeDefault = getDefault(field)
+      val maybeDefault = field.instance.getDefault
```

Comment:
I _think_ this might not be covered by tests.

Previously this would be `Option[Document]`, now it's `Option[A]`... if that doesn't cause an error I'm not sure it's even being exercised.

Reply:
I can double check either way, but `getDefault` returns an `Option[Document]` while `getDefaultValue` returns an `Option[A]`